client/ayon_core/hosts/maya/plugins/publish/collect_animation.py

```diff
@@ -17,7 +17,7 @@ class CollectAnimationOutputGeometry(pyblish.api.InstancePlugin):
 
     order = pyblish.api.CollectorOrder + 0.4
     families = ["animation"]
-    label = "Collect Animation Output Geometry"
+    label = "Collect Animation"
```

Comment:
```suggestion
    label = "Collect Animation Output Geometry"
```

With the collect user defined attributes logic removed from here again I think we can just leave this file untouched and remove the changes?